completions/puppet

```diff
@@ -101,16 +101,16 @@ _puppet()
             ;;
         puppet)
             case ${words[1]} in
-                agent|apply|cert|describe|doc|filebucket|kick|lookup|master|parser|queue|resource)
+                agent|apply|catalog|cert|describe|doc|epp|filebucket|help|kick|lookup|man|master|module|node|parser|report|resource)
                     subcommand=${words[1]}
```

Comment:
Well, perhaps all these can be squashed to one commit after all (I can take a look at it on merge), so please disregard the split comment above.